### DISCUSSION TITLE
How to change patroni.yml and /data/patroni/postgresql.conf ?

### DISCUSSION THREAD

**Discussion by AleksandrMalinovskiy** [autobase-tech/autobase#217] [Q&A]
Good afternoon. How to change patroni.yml and /data/patroni/postgresql.conf in a running cluster?

**Comment by vitabaks** [comment]
use  `patronictl edit-config`

**Reply by vitabaks** [reply]
https://github.com/vitabaks/postgresql_cluster#maintenance

**Reply by vitabaks** [reply]
in a future version of the playbook, we will automate the configuration change in DCS using ansible.
